onnxruntime/core/optimizer/qdq_transformer/selectors_actions/shared/utils.cc

```diff
@@ -131,7 +129,8 @@ static const OpVersionsAndSelector::OpVersionsMap GetGemmOpVersionsMap() {
 }
 static const OpVersionsAndSelector::OpVersionsMap GetInstanceAndLayerNormalizationOpVersionsMap() {
   return {{"InstanceNormalization", {}},
-          {"LayerNormalization", {}}};
+          {"LayerNormalization", {}},
+          {"RMSNormalization", {}}};
```

Comment:
can you provide some background on why the additional op type was added here?

and if a new op type is added here, the name `GetInstanceAndLayerNormalizationOpVersionsMap` is no longer accurate.